README.md

```diff
@@ -113,7 +110,17 @@ estimator = Estimator(options={"shots": int(1e4), "seed": 42})
 # define ansatz and initial point
 ansatz = EfficientSU2(qubit_op.num_qubits)
 num_params = ansatz.num_parameters
-initial_point = 2 * np.pi * np.random.random(num_params)
+initial_point = [2.95891852, 1.46946667, 3.34553262, 
```

Comment:
I don't know - technically its not invalid but it seems to takeover the example. How about

`initial_point = [0.1] * ansatz.num_parameters`

Yes you can make it more random, but maybe that's just as good for the sample

Reply:
```suggestion
initial_point = 2 * np.pi * np.random.random(num_params)
```

Reply:
Oops, sorry I thought this was a different PR... Nevermind :sweat_smile:

Reply:
Seems like its in an odd state now. I would go with with some simple init point as I described above - one could use all zeros but I tend to avoid that as things used to get stuck there in the past.